### PR TITLE
ANDROAPP-5621-mobile-ui-Create-dropdown-component

### PR DESCRIPTION
**Pull request by siddh1004**

### CODE REVIEW

**Review by andresmr — APPROVED** [review]

File: common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            state = InputShellState.UNFOCUSED,
+            selectedItem = selectedItem,
+            onResetButtonClicked = {
+                selectedItem = null
+            },
+            onArrowDropDownButtonClicked = {
+                expanded = !expanded
+            },
+        )
+        DropdownMenu(
```
Comment (msasikanth) [inline common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt:43]: The way the `DropdownMenu` is configured right now, it moves the content by px or so when it's opened. When we move the dropdown into the component, let's wrap the content and dropdown inside a box and use dropdown offset to make sure they are overlap.

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (siddh1004) [reply common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt:43]: Hi @msasikanth For this release, it has been decided to keep the dropdown out of the compnent and open it via callback. The dropdown would be moved inside the component in the next release.

Comment (siddh1004) [reply common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt:43]: This relates to above comment.

Comment (msasikanth) [reply common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt:43]: Could you wrap up the dropdown menu and input it inside a box? Or else it's pushing the content down a bit every time it's opened in the sample.

Comment (msasikanth) [reply common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt:43]: Will be useful to showcase the proper usage when they are configuring it inside the app.

Comment (siddh1004) [reply common/src/commonMain/kotlin/org/hisp/dhis/common/screens/InputDropDownScreen.kt]: @msasikanth Fixed.

**Review by msasikanth — APPROVED** [review]

**Pull request merged**